controllers/tasks.js

```diff
@@ -334,38 +322,40 @@ const updateTaskStatus = async (req, res, next) => {
     let userStatusUpdate;
     const taskId = req.params.id;
     const { userStatusFlag } = req.query;
+    const { status } = req.body;
```

Comment:
is status is required field or option in body?

Reply:
Since it's optional we can do this
const status = req.body?.status;

Reply:
changed this to the above stated way.